application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -56,24 +60,28 @@ public static <S extends CharSequence> Optional<S> autocomplete(CharSequence pre
      *
      * @param prefix the prefix to give suggestion for
      * @param candidates all the possible suggestions
-     * @param errorMargin error margin of suggestions, error percentage / 100
      * @return collection of autocomplete suggestions
```

Comment:
`* @return collection of autocomplete suggestions` this text assumes context that is incorrect from the POV of `StringDistances`. rephrase to `the matches closest to the given prefix, limited to the given limit`